header/js/typeahead.js

```diff
@@ -99,4 +107,23 @@ function trackSearchEvent (context) {
 	document.body.dispatchEvent(tracking);
 }
 
+// This is purposefully quite rubbish to be small, fast and target only the most-problematic chars.
+function latinize (text) {
+	const replacements = {
+		'a': ['à', 'á', 'â', 'ä', 'ã', 'å'],
```

Comment:
i think merge this for now and then we can fix more properly later.

i don't suppose there's an easy way to keep the diacritics on the text that the users sees is there?